### PR TITLE
Deploy

### PR DESCRIPTION
**Pull request by fejiroofficial**
**What does this PR do?
This a pull request for configuring eslint, removing an if statement from code block and for changing the name of a variable.

**Description of Task to be completed?**
Have the following
- have a proper indentation on test files(question.spec.js & answer.spec.js)
- have the if statement removed from `GET /api/v1/questions` controller function.

**What are the relevant pivotal tracker stories?**
#159951517

### CODE REVIEW

File: .eslintrc [hunk 1]
```diff
@@ -5,26 +5,23 @@
         "commonjs": true,
         "es6": true
     },
-    "extends": [
-        "eslint:recommended",
-        "airbnb"
-    ],
+    "extends": "eslint:recommended",
```
Comment (Billmike) [inline .eslintrc]: Your `eslint` config doesn't extend `airbnb` style guide. Any reason why?

File: .slugignore [hunk 2]
```diff
@@ -0,0 +1 @@
+yarn.lock
```
Comment (Billmike) [inline .slugignore]: Add a new line here

File: .eslintrc [hunk 3]
```diff
@@ -5,26 +5,23 @@
...
+             2
         ],
         "linebreak-style": [
             "error",
             "windows"
         ],
         "quotes": [
             "error",
-            "double"
+            "single"
```
Comment (Billmike) [inline .eslintrc]: With `airbnb`, preferred quotes are double, not single.

File: test/controller/answers.spec.js [hunk 4]
```diff
@@ -7,57 +7,57 @@ import { allQuestions, answers } from '../../datastore/questions';
...
-					.post('/api/v1/questions/1/answers')
-					.send(validAnswer)
-					.end((err, res) => {
-						expect(res.status).to.equal(201);
-						expect(res.body.status).to.equal('successful');
-						expect(res.body.answer).to.be.an('object');
-						done();
-					});
-			});
+  // test answer controller
```
Comment (Billmike) [inline test/controller/answers.spec.js]: No need for the in-line comments.

File: test/controller/answers.spec.js [hunk 5]
```diff
@@ -7,57 +7,57 @@ import { allQuestions, answers } from '../../datastore/questions';
...
+  });
+
+  describe('Post an answer', () => {
+    const validAnswer = {
+      answer: 'why not ask google!'
+    };
+    
+    describe('when invalid data is passed', () => {
+      it('should not create an answer resource', (done) => {
+        const invalidAnswer = Object.assign({}, validAnswer, {
```
Comment (Billmike) [inline test/controller/answers.spec.js]: You could re-write this to be like this:
```javascript
const invalidAnswer = { ...validAnswer, answer: '' };
```
This is cleaner, and achieves the same thing

File: test/controller/questions.spec.js [hunk 6]
```diff
@@ -7,126 +7,110 @@ import { allQuestions, answers } from '../../datastore/questions';
...
+
+  describe('Post a single question', () => {
+    const validQuestion = {
+      questionTitle: 'What a question?',
+      questionDescription: 'i have a question here',
+    };
+
+    describe('when invalid data is passed', () => {
+      it('should not create a question resource', (done) => {
+        const invalidQuestion = Object.assign({}, validQuestion, {
```
Comment (Billmike) [inline test/controller/questions.spec.js]: Same thing is applicable here:
```javascript
const invalidQuestion = { ...validQuestion, questionTitle: '', questionDescription: '' }
```

**Review by Billmike — CHANGES_REQUESTED** [review]

**Comment by Billmike** [comment]
1. Your PR does not follow the laid out convention. The PR title doesn't conform to the standard.
2. You have 6 commits for this PR. You need to squash it into one commit. You can refer to [this](https://github.com/k88hudson/git-flight-rules)

File: test/controller/answers.spec.js (hunk 4 above)
Comment (fejiroofficial) [reply test/controller/answers.spec.js]: noted

File: test/controller/answers.spec.js (hunk 5 above)
Comment (fejiroofficial) [reply test/controller/answers.spec.js]: okay

File: test/controller/questions.spec.js (hunk 6 above)
Comment (fejiroofficial) [reply test/controller/questions.spec.js]: noted

File: .eslintrc (hunk 3 above)
Comment (fejiroofficial) [reply .eslintrc]: okay

**Pull request closed without merging**